crd-convert/src/main/java/io/strimzi/kafka/crd/convert/converter/AbstractSpecableConverter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.converter;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.strimzi.api.annotations.ApiVersion;
+
+import static java.util.Arrays.asList;
+
+public abstract class AbstractSpecableConverter<T extends HasMetadata> extends Converter<T> {
```

Comment:
Javadoc which resources this can apply to? Or at least the expectation that T has a `/spec/tolerations` etc.

Reply:
OK ...